rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionService.scala

```diff
@@ -149,6 +160,7 @@ class ReportsExecutionService (
         reportsRepository.getReportsWithLowestId match {
           case Full(Some((id, report))) =>
             logger.debug(s"Initializing the status execution update to  id ${id}, date ${report.executionTimestamp}")
+            this.idForCheck = id
```

Comment:
ah, because the param is also named "id", and if I don't qualify the other id, there is a name conflict